src/AppInstallerCommonCore/DateTime.cpp

```diff
@@ -6,7 +6,7 @@
 namespace AppInstaller::Utility
 {
     // If moved to C++20, this can be replaced with standard library implementations.
-    void OutputTimepoint(std::ostream& stream, const std::chrono::system_clock::time_point& time)
+    void OutputTimePoint(std::ostream& stream, const std::chrono::system_clock::time_point& time)

```

Comment:
Given that the underlying type is `time_point`, it made sense to me to use this capitalization